src/include/utime.h

```diff
@@ -71,8 +71,10 @@ class utime_t {
     tv.tv_sec = v.tv_sec;
     tv.tv_nsec = v.tv_nsec;
   }
-  explicit utime_t(const ceph::real_time& rt) {
-    ceph_timespec ts = real_clock::to_ceph_timespec(rt);
+  // conversion from ceph::real_time/coarse_real_time
+  template <typename Clock, typename std::enable_if_t<!Clock::is_steady>* = nullptr>
+  explicit utime_t(const std::chrono::time_point<Clock>& rt) {
+    ceph_timespec ts = Clock::to_ceph_timespec(rt);
```

Comment:
This might eventually break if we add more clocks, but I don't really see us doing that.